manifests/base/jupyter-rocm-minimal-notebook-imagestream.yaml

```diff
@@ -16,7 +16,7 @@ spec:
   tags:
     # N Version of the image
     - annotations:
-        opendatahub.io/notebook-software: '[{"name":"ROCm","version":"6.1"},{"name":"Python","version":"v3.9"}]'
+        opendatahub.io/notebook-software: '[{"name":"ROCM","version":"6.1"},{"name":"Python","version":"v3.11"}]'
```

Comment:
Also, AMD use `ROCm` (lower `m`) in their materials, https://www.amd.com/en/products/software/rocm.html . Is there a particular reason we don't want to follow it?